site2/docs/functions-overview-new.md

```diff
@@ -0,0 +1,285 @@
+---
+id: functions-overview
+title: Pulsar Functions overview
+sidebar_label: Overview
+---
+
```

Comment:
I would suggest adding a banner to indicate that this document is under construction.

```
> The section of the documentation is the new documentation of Pulsar Function. It is still under construction. If you would like to checkout the documentation of Pulsar Function. Please check out the old [Pulsar Functions](...) documentation.
```

Reply:
Done.